src/main/java/com/homework/Executor.java

```diff
@@ -0,0 +1,24 @@
+package main.java.com.homework;
+
+public class Executor {
+
+    Client Yura = new Client(new Request(), "Yura");
```

Comment:
Why do all these variables names start with a capital letter?

Reply:
Ohhhh, sorry it is was written on automatic, I changed it.